key-manager/src/main/scala/org/bitcoins/keymanager/WalletStorage.scala

```diff
@@ -304,6 +304,33 @@ object WalletStorage {
     }
   }
 
+  def changeAesPassword(
+      seedPath: Path,
+      oldPasswordOpt: Option[AesPassword],
+      newPasswordOpt: Option[AesPassword]): MnemonicState = {
+    decryptMnemonicFromDisk(seedPath, oldPasswordOpt) match {
+      case Left(err) => sys.error(err.toString)
+      case Right(decrypted) =>
+        val fileName = seedPath.getFileName.toString
+        val backup = seedPath.getParent.resolve(fileName + ".backup")
+        Files.move(seedPath, backup)
+
+        val toWrite = newPasswordOpt match {
+          case Some(pass) =>
+            decrypted.encrypt(pass)
+          case None =>
+            decrypted
+        }
+
+        Try(writeMnemonicToDisk(seedPath, toWrite)) match {
+          case Failure(exception) => throw exception
```

Comment:
We should probably add a `log.warn()` or `log.error()` saying that the backup seed can be found at the backup file path.

Reply:
Do we want to move the backup file back to the original seed in this case? I'm 50/50 on that. It seems dangerous in the sense that it makes it easier to "silently" fail changing the password. 

However you may want to write application logic to handle this case and revert to using the `oldPasswordOpt` without downtime. Thoughts?

Reply:
I think it's fine as is. Likely if we need the backup file something went wrong outside our control, we should probably just leave the file and not touch it